web/client-api/src/main/java/io/deephaven/web/client/api/LoginCredentials.java

```diff
@@ -0,0 +1,47 @@
+package io.deephaven.web.client.api;
+
+import jsinterop.annotations.JsOverlay;
+import jsinterop.annotations.JsPackage;
+import jsinterop.annotations.JsType;
+
+@JsType(namespace = JsPackage.GLOBAL, name = "Object", isNative = true)
+public class LoginCredentials {
```

Comment:
Not necessary yet, except as part of the larger picture for dh.CoreClient.

Reply:
Actually slightly necessary now, so login() can be used to wait for a connection.